src/back/spv/rosetta_tests.rs

```diff
@@ -0,0 +1,16 @@
+use std::{fs, path::Path};
+
+const TEST_PATH: &str = "test-data";
+
+fn rosetta_test(file_name: &str) {
+    let test_dir = Path::new(TEST_PATH);
+    let input = fs::read_to_string(test_dir.join(file_name)).unwrap();
+    let module: crate::Module = ron::de::from_str(&input).unwrap();
+
+    let spv = super::Writer::new(&module.header, super::WriterFlags::NONE).write(&module);
```

Comment:
let's try to write it without warnings, they are distracting:)

Reply:
Must have been late - or early. Had already staged fix for this, but apparently forgot to commit and push :)